app/lib/cost_calculator.rb

```diff
@@ -0,0 +1,28 @@
+module CostCalculator
+  class << self
+    def cost(type, object, vat)
+      case type
+      when :travel_and_waiting_total
+        travel_and_waiting_total(vat, object)
+      end
+    end
+
+    private
+
+    def travel_and_waiting_total(vat, object)
+      pricing = Pricing.for(object)
+      vat_rate = pricing.vat
+      total_without_vat = get_work_total(WorkItem.where(claim_id: object[:id], work_type: %w[travel waiting]), pricing)
+      total = vat ? total_without_vat * (1 + vat_rate) : total_without_vat
+      total.round(2)
+    end
+
+    def get_work_total(items, pricing)
```

Comment:
naming, avoid use of get/set

```suggestion
    def work_items_total(items, pricing)
```